src/utils/constants.js

```diff
@@ -0,0 +1,114 @@
+import eyeOnImg from '../assets/eye-on.svg';
+import eyeOffImg from '../assets/eye-off.svg';
+
+export const EMAILREGEX = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/i;
+export const PASSWORDREGEX = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/;
```

Comment:
변수명을 대문자만으로 작성하니 가독성이 좋지 않죠!?
`SCREAMING_SNAKE_CASE`로 작성하시면 됩니다.
```suggestion
export const EMAIL_REGEX = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/i;
export const PASSWORD_REGEX = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/;
```